charts/console/templates/_helpers.tpl

```diff
@@ -100,6 +135,43 @@ Name of the conduktor license
 {{- end -}}
 {{- end -}}
 
+
+{{/*
+Name of the platform Service
+*/}}
+{{- define "conduktor.platform.serviceName" -}}
+    {{ include "common.names.fullname" . }}
+{{- end -}}
+
+{{/*
+Platform service internal domain name
+*/}}
+{{- define "conduktor.platform.serviceDomain" -}}
+{{- printf "%s.%s.svc.%s" (include "conduktor.platform.serviceName" .) .Release.Namespace .Values.clusterDomain -}}
+{{- end -}}
+
+{{/*
+Platform external url
+*/}}
+{{- define "conduktor.platform.externalUrl" -}}
+{{- $proto := ternary "https" "http" .Values.ingress.tls -}}
+{{- printf "%s://%s" $proto .Values.ingress.hostname -}}
+{{- end -}}
```

Comment:
You should also consider `.config.tls.selfSigned` & `.config.tls.existingSecret`

Reply:
Yes will do that